timesketch/frontend/dist/index.html

```diff
@@ -1 +1,5 @@
-<!DOCTYPE html><html lang=en><head><meta name=csrf-token content="{{ csrf_token() }}"><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/dist/favicon.ico><title>Timesketch</title><link href=/dist/css/chunk-common.73b6326f.css rel=preload as=style><link href=/dist/css/chunk-vendors.52c8c6fd.css rel=preload as=style><link href=/dist/js/chunk-common.1d17c35a.js rel=preload as=script><link href=/dist/js/chunk-vendors.1af0e4a6.js rel=preload as=script><link href=/dist/js/index.0c45ccdf.js rel=preload as=script><link href=/dist/css/chunk-vendors.52c8c6fd.css rel=stylesheet><link href=/dist/css/chunk-common.73b6326f.css rel=stylesheet></head><body><div id=app></div><script src=/dist/js/chunk-vendors.1af0e4a6.js></script><script src=/dist/js/chunk-common.1d17c35a.js></script><script src=/dist/js/index.0c45ccdf.js></script></body></html>
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
and you need to generate these again, we still got the <<<< HEAD from the merge conflicts

Reply:
damn it. fixed now. :)